url_shortener/config.py

```diff
@@ -5,7 +5,7 @@
 
 REDIS_PREFIX = os.getenv("REDIS_PREFIX", "short:")
 REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
-REDIS_PORT = int(os.getenv("REDIS_PORT", "6379"))
```

Comment:
I couldn't figure out a better way to do this. This was done quite quickly :) The long of it is that Docker 1.9+ going forward is deprecating the use of ``REDIS_PORT`` and ``REDIS_PORT_6379`` and other related env vars when linking containers (*AFAIK*) in favour of just populating the associated hosts files in the other container.